ttnn/cpp/ttnn/operations/eltwise/ternary_backward/device/ternary_backward_op.cpp

```diff
@@ -26,9 +28,9 @@ std::vector<Tensor> _addcmul_bw(
     const MemoryConfig& output_mem_config) {
     std::vector<Tensor> grad_tensor;
```

Comment:
We can create it with size 3 to avoid a few small allocations.
or even just do `return {grad, grad_a, grad_b};. (and use std::move here)` or you create grad_tensor with size 3 and:
grad_tensor[0] = grad;
and etc.